lib/agent0/agent0/hyperdrive/exec/execute_agent_trades.py

```diff
@@ -68,7 +61,9 @@ async def async_execute_single_agent_trade(
     # TODO preliminary search shows async tasks has very low overhead:
     # https://stackoverflow.com/questions/55761652/what-is-the-overhead-of-an-asyncio-task
     # However, should probably test what the limit number of trades an agent can make in one block
-    wallet_deltas_or_exception: list[HyperdriveWalletDeltas | BaseException] = await asyncio.gather(
+    wallet_deltas_or_exception: list[
+        Tuple[HyperdriveWalletDeltas, ReceiptBreakdown] | BaseException
```

Comment:
are you sure you need `Tuple` from typing here? I think `tuple[` will work as long as `annotations` is imported.

Reply:
Confirmed, you can replace `Tuple` with `tuple` in this entire file and remove the typing import `Tuple` because we have `from __future__ import annotations`. In python 3.11 this is the expected behavior, and we no longer need to import `annotations`.